ttnn/cpp/ttnn/operations/reduction/generic/device/kernels/dataflow/reader_unary_transpose_wh_interleaved_input_cols_partitioned.cpp

```diff
@@ -17,6 +17,7 @@ void kernel_main() {
     constexpr uint32_t Ht = get_compile_time_arg_val(1);
     constexpr uint32_t Wt = get_compile_time_arg_val(2);
     constexpr uint32_t HtWt = get_compile_time_arg_val(3);
+    constexpr uint32_t row_chunk = get_compile_time_arg_val(4);
 
     constexpr uint32_t cb_id_in0 = 0;
```

Comment:
It'd be good to change 0 to tt::CBIndex::c_0 and the 2 on line 29 below to tt::CBIndex::c_2